### PR TITLE
Systemd support

### PR DESCRIPTION
**Pull request by alfredodeza**
this correctly starts both the chacra service and celery using systemd.

Will restrict where we deploy (Ubuntu will need Xenial at the bare minimum)

Removes all circus related files.

### CODE REVIEW

**Comment by andrewschoen** [comment]
:thumbsup:

File: deploy/playbooks/roles/common/tasks/systemd.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+
+- name: ensure /var/log/celery dir exists
+  sudo: true
+  file: path=/var/log/celery state=directory owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} recurse=yes
+
+- name: ensure /etc/conf.d/ dir exists
+  sudo: true
+  file: path=/etc/conf.d state=directory
```
Comment (ktdreyer) [inline deploy/playbooks/roles/common/tasks/systemd.yml]: Typically this environment file would be `/etc/sysconfig/chacra` on RHEL, or `/etc/default/chacra` on Ubuntu

**Comment by ktdreyer** [comment]
Looks good! Just one issue with `/etc/conf.d/{{ app_name }}-sysconfig` (I think `/etc/default/chacra` would be better)

File: deploy/playbooks/roles/common/tasks/postgresql.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@
   sudo: yes
   lineinfile:
      # TODO: should not hardcode that version
-     dest: /etc/postgresql/9.3/main/pg_hba.conf
+     dest: /etc/postgresql/9.5/main/pg_hba.conf
```
Comment (dmick) [inline deploy/playbooks/roles/common/tasks/postgresql.yml]: Seems wrong that we're changing to a generic postgresql package dependency but keeping a specific (and different) version in this path.  I see the TODO, but, isn't there either a generic link without version or a way to query the installed version?

Comment (alfredodeza) [inline deploy/playbooks/roles/common/tasks/postgresql.yml]: I couldn't find a generic link but if there is a better option I wouldn't mind updating it.

**Pull request merged**